keras_nlp/samplers/beam_sampler_test.py

```diff
@@ -42,11 +43,18 @@ def join_as_string(self, x):
         return ["".join([self.int_lookup[i] for i in s]) for s in x.numpy()]
 
     def test_stateless_call(self):
-        def next(prompt, state, index):
+        def next(prompt, cache, index):
+            # Dummy hidden states.
+            hidden_states = tf.ones([10])
```

Comment:
I think for these dummy example we should still show a 2d shape with a batch dimension for hidden state. So ones/zeros with shape `(self.batch_size, 1)`, for example.

Reply:
sg